assets/blocks/view-certificate-link/view-certificate-link-edit.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps, Warning } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Course Overview block edit component.
+ *
+ * @param {Object} props
+ * @param {Object} props.context          Block context.
+ * @param {Object} props.context.postType Post type.
+ */
+const ViewCertificateLinkEdit = ( { context: { postType } } ) => {
+	const blockProps = useBlockProps();
+
+	if ( ! [ 'course', 'lesson' ].includes( postType ) ) {
```

Comment:
I didn't test, but did you try to use the [`ancestor`](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/nested-blocks-inner-blocks/#child-innerblocks-parent-and-ancestors) passing the name of the variation `sensei-lms/course-list`? I think the `anscestor` wouldn't have compatibility with all of our versions, but if it works, maybe it's worth to use?

Reply:
Done in [22a0768](https://github.com/woocommerce/sensei-certificates/pull/327/commits/22a0768969ce562b53347e13a5b333cda3f841fb). I left the manual warning in there as well for the older versions.